scipy/stats/_stats_py.py

```diff
@@ -9531,3 +9533,114 @@ def rankdata(a, method='average', *, axis=None, nan_policy='propagate'):
         result[nan_indexes] = np.nan
 
     return result
+
+
+def expectile(a, alpha=0.5, *, weights=None):
+    r"""Compute the expectile.
+
+    Expectiles are a generalization of the expectation in the same way as
+    quantiles are a generalization of the median. The expectile at level
+    `alpha = 0.5` is the mean (average). See Notes for more details.
+
+    Parameters
+    ----------
+    a : array_like
+        Input array or object that can be converted to an array.
+    alpha : float, default=0.5
+        The level of the expectile; `alpha=0.5` gives the mean.
+    weights : array_like, optional
+        The sample or case `weights` array must be broadcastable to the same
+        shape as `a`. Default is None, which gives each value a weight of 1.0.
+        An integer valued weight element acts like repeating the corresponding
+        observation in `a` that many times. See Notes for more details.
+
+    Returns
+    -------
+    expectile : ndarray
+        The empirical expectile at level `alpha`.
+
+    See Also
+    --------
+    numpy.mean : Arithmetic average
+    numpy.quantile : Quantile
+
+    Notes
+    -----
+    In general, the expectile at level `alpha` (:math:`\alpha`) of a
+    probability distribution :math:`F` with :math:`X\sim F` is given by the
+    unique solution :math:`t` of:
+
+    .. math::
+
+        \alpha E((X - t)_+) = (1 - \alpha) E((t - X)_+) \,.
+
+    Here, :math:`(x)_+ = \max(0, x)` is the positive part of :math:`x`.
+
+    The empirical expectile of a sample :math:`a_i` (the array `a`) is
```

Comment:
```suggestion
    The empirical expectile at level :math:`\alpha` (`alpha`) 
    of a sample :math:`a_i` (the array `a` ) is
```
To keep it consistent what is in the parentheses (math or code)